bookkeeper-stats-providers/twitter-ostrich-provider/pom.xml

```diff
@@ -32,6 +32,38 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.libdir>${basedir}/lib</project.libdir>
   </properties>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
@sijie sure I would like to move forward, let's clean up the poms at the end of the work, we are on the same page